modules/seqexec/server/src/main/scala/seqexec/server/SeqexecFailure.scala

```diff
@@ -3,12 +3,10 @@
 
 package seqexec.server
 
-import cats.ApplicativeError
-import cats.effect.IO
 import edu.gemini.seqexec.odb.SeqFailure
 import org.http4s.Uri
 
-sealed trait SeqexecFailure extends Product with Serializable
+sealed trait SeqexecFailure extends Throwable with Product with Serializable
```

Comment:
Can we extend `Exception` or `IOException` or something more fine-grained than `Throwable`?

Reply:
Yeah. it could be `Exception` not `IOException` as it is not always IO related